src/config/config.go

```diff
@@ -32,7 +32,7 @@ type ApiConfig struct {
 	Bind      string              `toml:"bind" json:"bind"`
 	BasicAuth *ApiBasicAuthConfig `toml:"basic_auth" json:"basic_auth"`
 	Tls       *ApiTlsConfig       `toml:"tls" json:"tls"`
-	Cors      bool		      `toml:"cors" json:"cors"`
+	Cors      bool                `toml:"cors" json:"cors"`
```

Comment:
My vim config automatically runs gofmt upon save. It must have done some tabs/space fix.